upgrading/upgrade-helm.adoc

```diff
@@ -7,7 +7,11 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can upgrade to the latest version of {product-title} from a supported older version. For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74.
+When upgrading, follow this guidance:
+
+* If Central is earlier than 3.74, you must upgrade to 3.74 before upgrading to a 4.x release. For upgrading Central from version 3.74 to 4.x, see the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74].
```

Comment:
Should this be for both Central/SecuredCluster ? 
```suggestion
* If the installed version is earlier than 3.74, you must upgrade to 3.74 before upgrading to a 4.x release. For upgrading from version 3.74 to 4.x, see the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74].
```

Reply:
Not sure, I assume so but I think @msugakov had specifically called out Central here, so maybe he can weigh in.

Reply:
Only Central. This is because of rocksdb->postgres database migrations.

@ludydoo if you are aware of the similar need for the Secured Cluster, **this** is a perfect time to mention that!

Reply:
I am not aware of any such thing

Reply:
I suppose we can close this thread as obsolete then.